package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nordcloud/gnui",
   "description": "Nordcloud Design System - a collection of reusable React components used in Nordcloud's SaaS products",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
I think we are adding a new functionality with this change and not fixing a bug, take a look how versioning works
https://semver.org/